.github/workflows/ci.yml

```diff
@@ -689,29 +689,56 @@ jobs:
           sudo apt-get update -y
           sudo apt-get install -y gcc-aarch64-linux-gnu
           echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+
       - name: Install Forc
         uses: actions-rs/cargo@v1
         with:
           command: install
           args: --debug --path ./forc
 
+      - name: Install Forc-Fmt
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: --debug --path ./forc-fmt
+
+      - name: Install Forc-LSP
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: --debug --path ./forc-lsp
+
+      - name: Install Forc-Explore
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: --debug --path ./forc-explore
+
       - name: Prep Assets
         id: prep_assets
         env:
           PLATFORM_NAME: ${{ matrix.job.platform }}
           TARGET: ${{ matrix.job.target }}
           ARCH: ${{ matrix.job.arch }}
         run: |
-          ZIP_FILE_NAME=forc.${{ env.PLATFORM_NAME }}_${{ env.ARCH }}.gz
+          mkdir -pv ./forc-binaries
+          for binary in forc forc-fmt forc-lsp forc-explore; do
+            cp $(which ${binary}) ./forc-binaries
+            gzip -c ./forc-binaries/$binary > ./forc-binaries/$binary.${{ env.PLATFORM_NAME }}_${{ env.ARCH }}.gz
```

Comment:
This looks like you're gzipping each binary individually. Would it not be better to gzip the entire directory at once (and can do it in a single `tar` command)?

Reply:
@adlerjohn No super super important reason. Just compressing the binaries so the files are smaller and the downloads are faster. Here's my back of the napkin match on the compression. And the compression doesn't really cost anything except 1 additional bash command 😄 

| Binary | Size uncompressed(Bytes) | Size compressed(Bytes) | Space saved |
| --------------- | --------------- | --------------- | ---- |
| forc | 16629856 | 6360527 | 62% |
| forc-fmt | 4723488 | 1756713 | 63% |
| forc-lsp | 8316312 | 2854229 | 66% |
| forc-explore | 8153208 | 2854229 | 65% |

Reply:
Ah, my concern was less about compressing period and more about compressing each binary individually. It's more steps for the end users and no compression gain since you can't compress twice. You should compress the entire directory at once.

Removing this line here should be sufficient to avoid compressing each binary individually.